sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntity.cs

```diff
@@ -127,17 +58,17 @@ private static bool IsEntitySource(HealthcareRelationInternal healthcareRelation
         public int Offset { get; }
 
         /// <summary>
-        /// .
+        /// Gets the length of input document.
         /// </summary>
         public int Length { get; }
 
         /// <summary>
-        /// .
+        /// Get the list of data sources for the entity.
         /// </summary>
         public IReadOnlyCollection<EntityDataSource> DataSources { get; }
 
         /// <summary>
-        /// .
+        /// Gets the dictionary for related entity with mapped relation type for each.
```

Comment:
Maybe something like `Gets the entities and the relationship between the entities.`